frontend/src/pages/chat/chat.tsx

```diff
@@ -12,18 +12,18 @@ import styles from './styles.module.scss';
 
 const Chat: React.FC = () => {
   const navigate = useNavigate();
-  const { isSidebarShown, setIsSidebarShown } = useSidebarState();
+  const { isSidebarShown, setIsSidebarShow } = useSidebarState();
 
   const handleBackButtonPress = useCallback(() => {
     navigate(AppRoute.CHATS);
-    setIsSidebarShown(true);
-  }, [setIsSidebarShown, navigate]);
+    setIsSidebarShow(true);
+  }, [setIsSidebarShow, navigate]);
 
   return (
     <>
       <ChatSidebar
         isSidebarShown={isSidebarShown}
-        setIsSidebarShown={setIsSidebarShown}
+        setIsSidebarShow={setIsSidebarShow}
```

Comment:
A9 rule

Reply:
@v0ldemar01 it is already fixed
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/6ead26c5-1f64-4b24-8eda-2ab1fd6c35d0)

Reply:
<img width="630" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/43109766/d4c58cb7-b4cc-40b0-b39e-be500da7c710">

Reply:
Not yet

Reply:
done